### PR TITLE
add zh trans content/zh/docs/reference/setup-tools/kubeadm/generated/…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+<!-- 
+### Synopsis
+-->
+### 概要
+
+
+<!--
+This command is not meant to be run on its own. See list of available subcommands.
+-->
+此命令并不意味着可以单独运行。请参阅可用子命令列表。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config.md]:
```suggestion
此命令并非设计用来单独运行。请参阅可用子命令列表。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    <tr>
+      <td colspan="2">--rootfs string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      [EXPERIMENTAL] The path to the 'real' host root filesystem.
+      -->
+      [实验] 指向 '真实' 宿主机的根目录。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_kubelet_config.md]:
```suggestion
      [实验] 指向宿主机上的 '实际' 根文件系统的路径。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver-kubelet-client.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+<!--
+Generate the certificate for the API server to connect to kubelet, and save them into apiserver-kubelet-client.cert and apiserver-kubelet-client.key files.
+-->
+生成供 API 服务器连接 kubelet 的证书，并将其保存到 apiserver-kubelet-client.cert 和 apiserver-kubelet-client.key 文件中。
+
+<!--
+If both files already exist, kubeadm skips the generation step and existing files will be used.
+-->
+如果两个文件都已存在，则 kubeadm 跳过生成步骤，将使用现有文件。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver-kubelet-client.md]:
```suggestion
如果两个文件都已存在，则 kubeadm 将跳过生成步骤，使用现有文件。
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver-kubelet-client.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+  </tbody>
+</table>
+
+
+
+<!--
+### Options inherited from parent commands
+-->
+### 继承于父命令的选择项
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver-kubelet-client.md]:
```suggestion
### 继承于父命令的选项
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver-kubelet-client.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+    <tr>
+      <td colspan="2">--rootfs string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">
+      <!--
+      [EXPERIMENTAL] The path to the 'real' host root filesystem.
+      -->
+      [实验] 到 '真实' 主机根目录文件系统路径。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_certs_apiserver-kubelet-client.md]:
```suggestion
[实验] 指向宿主机上的 '实际' 根文件系统的路径。
```

**Comment by tanjunchen** [comment]
@tengqm  已经修改 谢谢

**Pull request merged**